### PR TITLE
Add schema environment variable for VDI study access

### PR DESCRIPTION
**Pull request by dmgaldi**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -177,6 +177,9 @@ services:
       # Job Settings
       JOB_CACHE_TIMEOUT_DAYS: ${JOB_CACHE_TIMEOUT_DAYS:-30}
 
+      # VDI study access setting
+      VDI_SCHEMA_SUFFIX: ${VDI_SCHEMA_LIFECYCLE:-dev_n} # e.g. dev_n, prod_s, qa_n
```
Comment (steve-fischer-200) [inline docker-compose.yml:181]: i think i like `VDI_SCHEMA_SUFFIX` better, because n/s is not a lifecycle

**Review by steve-fischer-200 — CHANGES_REQUESTED** [review]
i suggested a change to the property name

File: docker-compose.yml (hunk 1 above)
Comment (dmgaldi) [reply docker-compose.yml:181]: Oops, will fix

**Review by steve-fischer-200 — APPROVED** [review]
i suggested a change to the property name

**Review by ryanrdoherty — APPROVED** [review]

**Pull request merged**